test/xml.json

```diff
@@ -1,4 +1,8 @@
 {
+  "application": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><ApplicationProvisioningResponse><Application><ApplicationId>1</ApplicationId><ServiceType>Messaging-V2</ServiceType><AppName>Test Application</AppName><MsgCallbackUrl>http://a.com</MsgCallbackUrl></Application></ApplicationProvisioningResponse>",
```

Comment:
Just a note for the future you can always use the ' single quote or ` tick so you don't have to escape the " .  It makes it a lot easier 👍